Application/EdFi.Ods.Api/Container/Modules/ApplicationModule.cs

```diff
@@ -68,6 +69,10 @@ protected override void Load(ContainerBuilder builder)
                 .As<IApplicationModelConvention>()
                 .SingleInstance();
 
+            builder.RegisterType<IdentitiesControllerRouteConvention>()
```

Comment:
All these registrations should be in an Identities feature module, not registered all the time.